src/components/results/loadflow/load-flow-result-tab.tsx

```diff
@@ -197,12 +198,14 @@ export const LoadFlowResultTab: FunctionComponent<LoadFlowTabProps> = ({
         const existingFilterIndex = initialFilters.findIndex(
             (f) => f.column === 'limitType'
         );
-        const updatedFilters =
+        let updatedFilters = convertFilterValues(initialFilters, intl);
+        updatedFilters =
             existingFilterIndex !== -1 &&
-            initialFilters[existingFilterIndex]?.value.length !== 0
-                ? initialFilters
+            //@ts-ignore we know it's an array
+            updatedFilters[existingFilterIndex]?.value.length !== 0
```

Comment:
I kept the existing behaviour
`data.value` can be an array of string (ie in this case), and empty arrays are truthly

Reply:
`let updatedFilters = convertFilterValues(initialFilters, intl);
        let limitTypeFilter = initialFilters.find(
            (f) => f.column === 'limitType'
        );

        // If 'limitType' filter does not exist or its value array is empty, add the default one
        if (
            !limitTypeFilter ||
            !(limitTypeFilter.value as LimitTypes[]).length
        ) {
            updatedFilters.push({
                column: 'limitType',
                dataType: FILTER_DATA_TYPES.TEXT,
                type: FILTER_TEXT_COMPARATORS.EQUALS,
                value: limitTypeValues,
            });
        }`

what do you think of that, instead of ts-ignore comment ?

Reply:
I took the habbit to do only necessary changes when I code here as it's the guideline, but yeah it's much better so I take it :+1: